src/app/[locale]/settings/notifiers/RemoveNotifierButton.tsx

```diff
@@ -7,11 +7,12 @@ import { toast } from 'react-toastify';
 const RemoveNotifierButton = ({ id }: { id: number }) => {
   const remove = async () => {
     try {
-      await toast.promise(removeNotifier(id), {
-        pending: 'Removing notifier...',
-        success: 'Notifier removed!',
-        error: 'Failed to remove notifier'
-      });
+      confirm('Are you sure you want to delete this notifier?') &&
```

Comment:
Hardcoded or i18n? Technically none of the settings pages use i18n so maybe this is for a later fix?